src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -993,6 +1009,56 @@ protected internal virtual IDurableClient GetClient(DurableClientAttribute attri
             return client;
         }
 
+        /// <summary>
+        /// Gets a <see cref="IDurableClient"/> using configuration from a <see cref="DurableClientAttribute"/> instance.
+        /// </summary>
+        /// <param name="attribute">The attribute containing the client configuration parameters.</param>
+        /// <returns>Returns a <see cref="IDurableClient"/> instance. The returned instance may be a cached instance.</returns>
+        protected internal virtual T GetGeneratedClient<T>(DurableClientAttribute attribute)
+        {

```

Comment:
I'm not seeing any place use these protected methods? Are these leftover from a previous implementation, or am I missing the usage?

Reply:
Not used. Removed.